.factory/automation.yml

```diff
@@ -52,6 +52,7 @@ build:
         bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
         bazel test $(bazel query 'kind(kt_jvm_test, //...) except kind(kt_jvm_test, //test/...)') --test_output=errors
     test-integration:
+      machine: 8-core-32-gb
```

Comment:
Bumping this here as this makes test reliably pass. The default machines are 2 threads, which I think are too weak to reliably run Linux + JVM + Studio itself. 

I've tested this on another branch, running each test 10 times: https://github.com/jamesreprise/typedb-studio/commit/9fa9466840f7895443146a886126e460acf20dd1